### ISSUE TITLE
Warning when using with webpack

### ISSUE THREAD

**Issue by Fjandin** [cisco/node-jose#54]
I'm using node-jose with webpack and i'm getting this warning.
```
./~/node-jose/lib/algorithms/helpers.js
Critical dependencies:
53:13-29 the request of a dependency is an expression
 @ ./~/node-jose/lib/algorithms/helpers.js 53:13-29
```

**Comment by Fjandin** [comment]
https://github.com/cisco/node-jose/pull/56

**Comment by linuxwolf** [comment]
Sorry for the delayed responses.

The warning is likely because 'crypto' is imported using a statement, which prevents browserify from including it without requiring descendants from knowing to exclude it.

**Comment by linuxwolf** [comment]
Fixed with #56.
